public/challenge-contributions/3/asavov.html

```diff
@@ -1,11 +1,108 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Challenge 3</title>
-</head>
-<body>
-  <h1>This is the template HTML file for this challenge.</h1>
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>VISA</title>
+    <style>
+      * {
+        border: 0;
+        padding: 0;
+        margin: 0;
+        box-sizing: border-box;
+        font-family: Arial, Helvetica, sans-serif;
+        color: white;
+        line-height: 1;
+      }
+      img {
+        display: block;
+        margin: auto;
+        max-width: 100%;
+        max-height: 100%;
+        border-radius: 7px;
+        -webkit-box-shadow: 0px 0px 5px -1px rgba(0, 0, 0, 1);
+        -moz-box-shadow: 0px 0px 5px -1px rgba(0, 0, 0, 1);
+        box-shadow: 0px 0px 5px -1px rgba(0, 0, 0, 1);
+      }
+      figure {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        position: relative;
+      }
+      .wrapper {
+        width: 100vw;
+        height: 100vh;
+        margin: auto;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+      }
+      .absolute {
+        position: absolute;
+      }
+      .fig-top-left {
+        top: 10px;
+        left: 20px;
+      }
+      .fig-top-right {
+        top: 10px;
+        right: 20px;
+      }
+      h1 {
+        font-size: 61.04px;
+      }
+      h4 {
+        font-size: 47.78px;
+      }
+      .fig-bottom-right {
+        bottom: 10px;
+        right: 20px;
+      }
+      .fig-bottom-left {
+        bottom: 20px;
+        left: 20px;
+      }
+      .fig-center {
+        top: 50%;
+        right: 20px;
+      }
+      .scene {
+        transform: perspective(600px);
+      }
+      .front,
+      .back {
+        backface-visibility: hidden;
+      }
+      .back {
+        transform: rotateY(180deg);
+      }
+      .scene:hover figure {
+        transition: transform 600ms ease;
+        transform-style: preserve-3d;
```

Comment:
You don't have your 3d working correctly. You are almost there, try to move this line to line 71 and increase the prespective to higher value (maybe 1000).

Reply:
I confirm that 3d is working 👍🏻